jedi_vim.py

```diff
@@ -262,10 +263,13 @@ def goto(mode="goto", no_output=False):
                                  for _ in range(4)])
                         f.write('{0}\t{1}\t{2}'.format(tagname, d.module_path,
                             'call cursor({0}, {1})'.format(d.line, d.column + 1)))
-                        f.seek(0)
```

Comment:
@wilywampa @blueyed 
Do you guys know why this was there in the beginning? Why would you seek there?